tests/etl/test_bugzilla.py

```diff
@@ -53,19 +74,21 @@ def test_bz_reopen_bugs(request, mock_bugzilla_reopen_request, client, test_job,
             {
                 "status": "REOPENED",
                 "comment": {
-                    "body": "New failure instance: https://treeherder.mozilla.org/logviewer?job_id=1&repo=mozilla-central"
+                    "body": "New failure instance: https://treeherder.mozilla.org/logviewer?job_id=5&repo=mozilla-central"
```

Comment:
Understanding the returned test data based on the code generation took some time.